index.md

```diff
@@ -17,8 +17,8 @@ large number of consumers. Pulp can locally mirror all or part of a repository,
 software packages in repositories, and manage many types of content from multiple sources in one
 place.
 
-Pulp has a [REST API](https://docs.pulpproject.org/dev-guide/integration/rest-api/index.html) and
-[command line interface](https://docs.pulpproject.org/user-guide/admin-client/index.html) for
+Pulp has a [REST API](https://docs.pulpproject.org/en/2.21/dev-guide/integration/rest-api/index.html) and
+[command line interface](https://docs.pulpproject.org/en/2.21/user-guide/admin-client/index.html) for
```

Comment:
I was guessing the links broke because of the release, in which case all of the above comments to change to 3.0 relevant links is the right move, but I left 2.21 links for now on my initial PR all because of this one line.
We don't have a command line interface/equivalent pages per se, so should this be removed or do we want to link to something else here?

Do we want to call out the changes from 2 to 3 on the main page for folks coming to check it out right now?
https://docs.pulpproject.org/from-pulp-2.html

Reply:
This command-line reference should be removed.

We should put a link to the changes from pulp 2 on our website.